src/components/Shared/UserProfile.tsx

```diff
@@ -16,18 +16,22 @@ interface Props {
   profile: Profile;
   showBio?: boolean;
   showFollow?: boolean;
+  showHandle?: boolean;
   followStatusLoading?: boolean;
   isFollowing?: boolean;
   isBig?: boolean;
+  messagePreview?: string;
```

Comment:
**nitpick:** looks like a not good idea to add `messagePreview` in this component

Devs won't understand what it does when they are using this component somewhere

Reply:
Yeah I was going back and forth on creating a new component similar to this (but stripped down) specifically for messages or not. I'll pull it out into its own unless you have other ideas!

Reply:
It's good creating new component for this 😊